### ISSUE TITLE
Is this SDK support 3D Secure v2 currently?

### ISSUE THREAD

**Issue by ylvict** [globalpayments/dotnet-sdk#6]
Just as issue title.

We are trying to implement 3DS v2 support in our website, but not sure if this SDK support 3DS v2 natively.

**Comment by ylvict** [comment]
@securesubmit-buildmaster 
Is there any update about 3DS (3D Secure) v2 support for this dotnet SDK?

**Comment by slogsdon** [comment]
@ylvict Apologies for the delay. We are currently working through 3DS 2.0 support. This work includes changes on the API and HPP servers, client libraries, and server-side SDKs.

**Comment by ylvict** [comment]
Thanks @slogsdon , looking forward to the support.

**Comment by ylvict** [comment]
Hi @slogsdon, Is there a timeline or schedule info about the 3DS v2 supporting progress? we`re really looking forward to that.

**Comment by slogsdon** [comment]
@ylvict We're in active development, but I do not have an ETA at this time.

**Comment by liamgold** [comment]
@slogsdon is it likely to be released close to September or earlier?

**Comment by slogsdon** [comment]
@liamgold Yes, we will have a solution available before September. We are working internally to get a better estimate for when the backing services and SDKs/libraries will be ready for developers to begin the integration work.

**Comment by martpendle** [comment]
Thanks @slogsdon please keep us updated on the progress of this. Can you tell us what state the currently available code is in? i.e. are some features implemented and ready for beta testing?

**Comment by martpendle** [comment]
Thanks @slogsdon please keep us updated on the progress of this. Can you tell us what state the currently available code is in? i.e. are some features implemented and ready for beta testing?

**Comment by ylvict** [comment]
@slogsdon Is there any update for the progress of 3ds v2 support? we`re really worried about SCA change in September.

**Comment by jonathancounihan** [comment]
Any update on this ticket please? When will 3DS V2 be supported via the SDK?

**Comment by seanmacdomhnall-gp** [comment]
Hi @jonathancounihan please see the step by step guide here: https://developer.globalpay.com/ecommerce/3d-secure-version2#hpp
